core/platform/auth/firebase_auth_services.py

```
@@ -184,6 +185,100 @@ def get_all_external_accounts() -> List[auth_domain.ExternalAccount]:
+    """Deletes multiple external accounts from Firebase.
+
+    PERF: This operation uses batching to fire less network requests.
+
+    Args:
+        accounts: Iterable[auth_domain.ExternalAccount]. The accounts to delete.
+
+    Raises:
+        auth_domain.AuthProviderError. If any accounts failed to be deleted.
+    """
+    errors = [
+        'At index=%d: %s' % (batch_offset + error_info.index, error_info.reason)
```

Ruff (0.14.14)

[warning] Use format specifiers instead of percent format

(UP031)